quarto-render/README.md

```diff
@@ -0,0 +1,103 @@
+# A Github Action to Render Quarto Documents
+
+This repository stores the `quarto-render` [GitHub Action](https://github.com/features/actions) for use with [Quarto](https://quarto.org/) projects. This action will call `quarto render` on the root directory of the repo. It also has example workflows for some common uses centered around web publishing.
```

Comment:
> It also has example workflows for some common uses centered around web publishing.

Do you want to move the examples in this repo ? We could make a example sections, like in `r-lib/actions` repo. 

Or have you moved them elsewhere already ?